sources/metadata/everest/1.0.0-rc1.yaml

```diff
@@ -0,0 +1,3 @@
+version: 1.0.0-rc1
+supported:
+  cli: '>= 1.0.0'
```

Comment:
It's okay to exclude the metadata tag here, since the CLI checks only the core version when verifying